src/custom.css

```diff
@@ -0,0 +1,7 @@
+.btn:hover {
```

Comment:
why is there need to create a separate file when index.css exists

Reply:
ok i will set it in the index.css file